src/App.js

```diff
@@ -32,7 +38,15 @@ export default class App extends Component {
            </select>
         </div>
         <ImageList images={filteredImages}/>
+        <div>
+          <select className="unicornFilter" onChange={handleChange}>
```

Comment:
I would make your <select> tag sections into their own components. Usually you want App.js to be pretty clean and simple with all of your html in different components. Looks awesome though!